### PR TITLE
Update with @stacks.js/* and various minor improvements

### PR DESCRIPTION
**Pull request by markmhx**
- Integrate `@stacks/auth` instead of `@blockstack/connect`
- Reduce Stacks-related code duplication with new `stacks` module
- Improve README with new screenshot
- Normalize naming around "Todos" instead of "To-do list" etc.
- Broaden scope of homepage tagline
- Simplify package name and add author
- Fix minor React-related console warnings

This PR resolves / supplants https://github.com/blockstack/todos/pull/53 and is needed mainly to update content found at https://docs.blockstack.org/authentication/building-todo-app to see how Stacks.js is used to integrate authentication.

### CODE REVIEW

**Review by yknl — APPROVED** [review]
LGTM

File: src/stacks.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { showBlockstackConnect } from '@stacks/connect';
+import { AppConfig, UserSession } from '@stacks/auth';
+
+const appConfig = new AppConfig();
+const userSession = new UserSession({ appConfig });
+
+function authenticate(sendToSignIn) {
```
Comment (kyranjamie) [inline src/stacks.js:7]: We tend to follow inline export style

```suggestion
export const userSession = new UserSession({ appConfig });

export function authenticate(sendToSignIn) {
```

(and remove export at bottom)

Comment (markmhx) [reply src/stacks.js]: Thanks, good tip 👍

**Comment by markmhx** [comment]
@kyranjamie Mind reviewing again with particular focus on the `stacks.js` module? I've refactored more of the `@stacks/*` imports into it so that we can reference them all in one place from the docs and keep things simpler for readers.

**Comment by kyranjamie** [comment]
I can, though following [discussion here](https://paper.dropbox.com/doc/Stacks-Connect-Package-xcOoRpOz90lHDWciJ62Ej), maybe we want to wait before merging this if we're going to change the packaging structures so that connect exports auth, rather than this way around.

**Comment by markmhx** [comment]
I'd prefer not to make that packages discussion blocking here since the related documentation is currently broken and need these changes urgently to fix them.

**Review by kyranjamie — APPROVED** [review]
LGTM @markmhx 

I'd be inclined to rename `src/stacks.js` to `stacks-auth` or something to be a bit more descriptive what it contains, otherwise looks clean.

**Comment by markmhx** [comment]
@kyranjamie I've improved the relevant `src` libraries into `auth.js` and `storage.js`. Mind taking another look?

**Comment by kyranjamie** [comment]
All good

**Pull request merged**